pkg/webhooks/pod_admission_webhook.go

```diff
@@ -87,7 +87,7 @@ func (p *podWebhook) leaderPodForFollower(ctx context.Context, pod *corev1.Pod)
 	log := ctrl.LoggerFrom(ctx)
 	leaderPodName, err := genLeaderPodName(pod)
 	if err != nil {
-		log.Error(err, "getting leader pod name for follower pod")
+		log.V(3).Info("leader pod (%s) for follower pod does not exist or has not been scheduled yet", leaderPodName)
```

Comment:
this is not where we lookup the leader, this is just generating its name. The error for looking up the leader happens below at line 102

Reply:
Updated. Let me know if you have any thoughts on the new error message. I thought about changing the upstream caller which logs the full error to log at INFO level instead, since multiple users have reached out thinking this was the problem when it was actually a problem with their Job template or something similiar, simply because the message is logged at ERROR level.

Example:

`ERROR	admission	finding leader pod for follower	{"webhookGroup": "", "webhookKind": "Pod", "Pod": {"name":"","namespace":"default"}, "namespace": "default", "name": "", .....  "error": "incorrect number of leader pods for this job (expected 1, got 0)"}`



For now though, I just changed the error message to mention it is an expected, transient error and include the leader pod name.